js/layout-javascript/news-feed-code.js

```diff
@@ -1739,16 +1777,44 @@ DynamicList.prototype.addFilters = function(records) {
 
     var filtersData = {
       filtersInOverlay: _this.data.filtersInOverlay,
+      detailViewOptions: _this.data.detailViewOptions,
       filters: filters
     };
+
     var template = _this.data.advancedSettings && _this.data.advancedSettings.filterHTML
       ? Handlebars.compile(_this.data.advancedSettings.filterHTML)
       : Handlebars.compile(filtersTemplate());
 
     _.remove(filters, function(filter) {
       return _.isEmpty(filter.data);
     });
+
     _this.$container.find('.filter-holder').html(template(filtersData));
+    _this.datePickerFrom = _this.$container.find('#dateFrom');
+    _this.datePickerTo = _this.$container.find('#dateTo');
+
+    _this.datePicker = _this.$container.find('input.date-picker').datepicker({
+      format: 'yyyy-mm-dd', // specific date format using lower case for Bootstrap datepicker
+      todayHighlight: true,
+      autoclose: true
+    }).on('changeDate', function(e) {
+      _this.value = moment(e.date).format(_this.DATE_FORMAT);
+    });
+
+    _this.datePickerFrom.datepicker({
+      format: 'yyyy-mm-dd',
+      autoclose: true
+    }).on('changeDate', function(e) {
+      _this.datePickerTo.datepicker('setStartDate', new Date(e.date));
```

Comment:
https://bootstrap-datepicker.readthedocs.io/en/latest/methods.html#setstartdate

This looks like it's setting a limit to the the date picker. We specifically said multiple times that users should not be limited on which dates they can pick.